classes/class-ep-debug-bar-elasticpress.php

```diff
@@ -162,12 +166,84 @@ public function render() {
 						<?php endif; ?>
 						<a class="copy-curl" data-request="<?php echo esc_attr( addcslashes( $curl_request, '"' ) ); ?>">Copy cURL Request</a>
 					</li><?php
-					
+
 				endforeach;
 
 			?></ol><?php
 
 		endif;
 	}
 
+	/**
+	 * Display debug info that effects the Elastic Press logs being visible or not.
+	 */
+	protected function display_debugging_info() {
+		$is_debug_errors = false;
+		?>
+		<div class="qm-boxed">
+			<section>
+				<h3><?php esc_html_e( 'Debug Info', 'debug-bar' ); ?></h3>
+				<p>
+					<?php esc_html_e( 'Elastic Press query logs not being displayed can be due to one of the following reasons:', 'debug-bar' ); ?>
+				</p>
```

Comment:
This should go inside the check below to see if we should display debug settings. It won't make sense when there's nothing to show.

Reply:
Well I'm struggling a bit with generic table/note that makes sense if either of the checks I'm doing below is true. Hence originally I added a line saying all is good when all checks passed.

If I move this note into the first condition and it's ok, however, the later check aren't passed - this note will not be displayed and I think it does help to clarify what and why we're displaying. So not quire sure what to do here.

Reply:
I'm not sure that just because EP is currently syncing that it can't make and log queries, so I don't think there needs to be a generic note for this table.

The status as to whether it's indexing or not isn't an error or pass/fail type situation, as in my other note it's just a general status that's useful to show.

Reply:
From the following comment https://github.com/humanmade/fox-sports-asia/pull/1636#issuecomment-505121665 I understood that when indexing, the queries are run by standard WP and not by Elastic Press, hence, during the time of indexing the logging of queries won't work as query is all WP.
Is that right?

Reply:
I see. So yes that may well be the case - it doesn't stop this plugin from logging though if any EP requests are successfully made but I see what you mean now. Sorry I was missing that bit of context!

So your generic note at the top of the table makes sense now, we just want the table to only show if the constants aren't set or it's currently indexing.